lib/diagnostic.md

```diff
@@ -3,6 +3,12 @@
 Using `inspect element`, write four selectors found on the main page for this GitHub repo.
 
 <!-- your answer starts here -->
+```<div class="pagehead repohead instapaper_ignore readability-menu experiment-repo-nav">
+  <div class="container repohead-details-container">
+<ul class="pagehead-actions">
+ <div id="start-of-content" class="accessibility-aid"></div>
+ ```
+
```

Comment:
this is a selector: start-of-content
this is a div: <div id="start-of-content"></div>